content/teaching-curriculum/introducing-teach-like-a-champion.md

```diff
@@ -0,0 +1,29 @@
+---
+_db_id: 
+content_type: topic
+flavours:
+- teaching
+learning_outcomes: null
+prerequisites:
+  hard: []
+  soft: []
+ready: true
+tags: 
+title: Introducing Teach Like A Champion
+---
+
+Welcome to our 
+
```

Comment:
This is a little strange. It will get rendered as:

```
<h1>Introducing Teach Like A Champion</h1>
<p>Welcome to our </p>
<h2>Intro to Teach like a Champion<h2/>
<h2>Quick Familiarty Projects</h2>
```

Better would be 

```
<h1>Introducing Teach Like A Champion</h1>
<h2>Quick Familiarty Projects</h2>
```